### PR TITLE
Use uid/gid from /srv/magento to  recreate the www-data user

### PR DESCRIPTION
**Pull request by mrphang**

### CODE REVIEW

File: apache/Dockerfile [hunk 1]
```diff
@@ -21,8 +21,8 @@ RUN apt-get -qqy update \
                            zip \
  && a2enmod rewrite \
  && a2ensite magento.conf \
-  # Idiopathic rm failure; loop is workaround
- && until rm -rf /var/lib/apt/lists; do sleep 1; done
+ # Idiopathic rm failure; loop is workaround
+ && until [ -Z $RESULT ]; do rm -rf /var/lib/apt/lists && RESULT=1; sleep 1; done
```
Comment (kojiromike) [inline apache/Dockerfile]: why?

**Comment by kojiromike** [comment]
Resolved by b0fef74 and 0db1944

File: apache/Dockerfile (hunk 1 above)
Comment (mrphang) [inline apache/Dockerfile]: It might've been an issue with my system. but It seemed like it was ignoring the -rf flags.

Comment (kojiromike) [inline apache/Dockerfile]: It wasn't being ignored; there is a general issue with aufs and rm at times. In any case, the loop resolves it. (Notice that docker doesn't just abort at this step.)

Regardless, this is a separate problem. If it needs to be resolved further, it should go in a different pull request.
